tests/integration/fns.sh

```
@@ -134,18 +140,24 @@ launch_aggregator() {
-     --tag "$name" \
+     --name "$name" \
      --config "$SCRIPT_DIR/lib/$name/config.yaml" \
      --pubkey-write-path "$SCRIPT_DIR/output/pubkey.bin" \
-     --plaintext-write-path "$SCRIPT_DIR/output/plaintext.txt" 2>&1 | tee >(strip_ansi > "$log_file") & echo $! > "/tmp/enclave.${ID}_${name}.pid"
+     --plaintext-write-path "$SCRIPT_DIR/output/plaintext.txt" $extra_args 2>&1 | tee >(strip_ansi > "$log_file") & echo $! > "/tmp/enclave.${ID}_${name}.pid"
 }
 
 kill_proc() {
   local name=$1
-  local pid=$(ps aux | grep 'enclave' | grep "\--tag $name" | awk '{ print $2 }')
+  local pid=$(ps aux | grep 'enclave' | grep "\--name $name" | awk '{ print $2 }')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)